packages/web/docs/src/app/federation/page.tsx

```diff
@@ -616,7 +617,7 @@ function WhyHive({ className }: { className?: string }) {
       <Heading as="h2" size="md" className="text-balance sm:px-6 sm:text-center">
         Why Choose Hive for GraphQL Federation?
       </Heading>
-      <ul className="flex flex-row flex-wrap justify-center divide-y divide-solid sm:mt-6 sm:divide-x sm:divide-y-0 md:mt-16 md:px-6 xl:px-16">
+      <ul className="flex flex-row flex-wrap justify-center divide-y divide-solid divide-gray-200 sm:mt-6 sm:divide-x sm:divide-y-0 md:mt-16 md:px-6 xl:px-16">
```

Comment:
Adding a border color is needed here, because Tailwind 4 doesn't change the default border color anymore, so this change _preserves_ how it looks in prod now.